package sayed;
import java.io.*;
import java.util.*;


public class Sayed {
              
	  
       public static void main (String [] args)
       {
    	   int[] arr   = new int[100];
    	   
           Random r = new Random();

      try{
    	
    	 Writer out = new FileWriter("E:/infile.txt");
    	  
    	  
    	 for(int i =0;i<20;i++)
    	 {
    		 int x;
    		x = r.nextInt(100);
    		  System.out.print(x+ " ");
    		out.write(x+"");
             out.write(" ");
    		
    	 }
    	 out.close();
    	  System.out.print("\n");
    	  
      }
      catch (IOException e)
      {
    	  System.out.println("here is an exception in first file");
      }
      
      try
      {
    	  Reader in = new FileReader("E:/infile.txt");
    	  
    	  int s;
    	  
    	  for(int i = 0;i<20;i++)
    	  {
    		  s = in.read();
    		  arr[i] = (int)s;
    		  System.out.print(s+ " ");
    	  }
    	  in.close();
    	  
      }
      
      catch (IOException e)
      {
    	  
      }
      
      for(int i =0;i<20;i++)
      {
    	  for(int j = i+1;j<20;j++)
    	  {
    		  if(arr[i]>arr[j])
    		  {
    			  
    			  int temp = arr[i];
    			  arr[i] = arr[j];
    			  arr[j] = temp;
    		  }
    	  }
      }
      
      try
      {
    	  Writer result = new FileWriter("E:/result.txt");
    	  
    	  for(int i =0;i<20;i++)
     	 {
     		
    		  result.write(Integer.valueOf(arr[i]) +  " ");
              
     		
     	 }
      	result.close();
      }
      catch(IOException e)
      {
    	System.out.println("here is an exception in result file");  
      }
}
}